ui-v2/app/components/tomography-graph.js

```diff
@@ -7,7 +7,7 @@ const inset = function(num) {
   return insetSize * num;
 };
 const milliseconds = function(num, max) {
-  return max > 0 ? parseInt(max * num) / 100 : 0;
+  return max > 0 ? parseFloat((parseInt(max * num) / 100).toFixed(2)) : 0;
```

Comment:
`parseFloat` here is superfluous since `parseInt(anything) / 100` is guaranteed to be a `Number` already.

Reply:
Nvm, just reread the code, `toFixed` results in a string, and `parseFloat` casts it back to a number.

If want you're interested in is two places of precision, you can avoid the typecasting completely:

```js
Math.round(max * num) / 100;
```

Reply:
Okay, one last comment. Why be picky about the decimal places here? If it's used for presentation, shouldn't it use the `format-number` helper? If it's not used for presentation, wouldn't you want the precision?